client/request_types.go

```diff
@@ -54,6 +55,14 @@ type BalanceReply struct {
 	Height int64 `json:"height"`
 }
```

Comment:
If we only allow validator owners using their own node to trigger allegation commands, need to confirm with Pavan, then we don't need the address in below request structs.

Reply:
VoteRequests address